### DISCUSSION TITLE
gsm for ARM64?

### DISCUSSION THREAD

**Discussion by mverissi** [grimme-lab/xtb#808] [Q&A]
Dear developers and users,

I want to set up a gsm calculation to find a reaction path. Since it's a small system, I'd like to run the calculation on my personal laptop, a Macbook with a M2 chip. Is there a build of gsm for ARM64? If not, how should I proceed?

Best regards,

Marcos

**Comment by epunzal2** [comment]
Have you tried these? You can use xTB methods as the Hamiltonian.
https://github.com/ZimmermanGroup/pyGSM
https://github.com/eljost/pysisyphus
